### ISSUE TITLE
Unlimited stock show as Out of Stock in ld+json

### ISSUE THREAD

**Issue by peteeveleigh** [nystudio107/craft-seomatic#812]
A Commerce variant marked as having unlimited stock is shown in the ld+json output as being out of stock.
Setting the stock level to a number switches the availability to show as in stock.

### To reproduce
Set a product variant to have unlimited stock. Observe the @graph/offers/availability value. It will show as 
`                "availability": "http://schema.org/OutOfStock",
`

### Expected behaviour
The value should show
`                "availability": "http://schema.org/InStock",
`

### Versions
- Plugin version: 3.3.29
- Craft version: Pro 3.5.17.1
- Commerce version: 3.2.14
--

**Comment by peteeveleigh** [comment]
OK this is weird.
I thought I'd have a dig around the plugin to see if I could figure out where the issue was.

This is working in my local dev environment.

Looking in `src/seomatic-config/productmeta/JsonLdContainer.php` I found the conditional that outputs either InStock or OutOfStock I couldn't see anything wrong with it so I changed this to always say InStock just to confirm this was the correct bit of code. Then I changed it  back to the original conditional.

Now it seems to work properly on all products. 🤷‍♂️ 

I can't make the same thing happen on the production site so not sure what I did to make it work locally.
